DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -241,6 +241,19 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 	Sdk->conference()
 	    .add_event_handler([this](const dolbyio::comms::conference_status_updated& Event)
 	                       { UpdateStatus(Event.status); })
+	    .then(
+	        [this](dolbyio::comms::event_handler_id)
+	        {
+		        return Sdk->conference().add_event_handler(
+		            [this](const dolbyio::comms::participant_added& Event)
+		            {
+			            const FDolbyIOParticipantInfo Info = ToUnrealParticipantInfo(Event.participant);
+						auto Status = Event.participant.status;
+			    		DLB_UE_LOG(Log, "Participant added event: UserID=%s Name=%s ExternalID=%s Status=%s",
+			                       *Info.UserID, *Info.Name, *Info.ExternalID, Status ? *ToString(*Status) : TEXT("not set"));
+						BroadcastEvent(OnParticipantAdded, Info);
```

Comment:
You would have to add status information to FDolbyIOParticipantInfo if you want to do it like this, because now there is no way to tell if that user is on_air or left. I would wrap this entire code with `if (status == on_air)` to keep current behavior (while also fixing the issue) and then remove that if when the 1:1 solution is ready.

Reply:
I had that if and it seemed to me I can simplify that. Thanks